Sources/_InternalTestSupport/BuildSystemProvider+Configuration.swift

```diff
@@ -34,4 +36,36 @@ extension BuildSystemProvider.Kind {
                 return ["apple", "Products" , "\(config)".capitalized + suffix]
         }
     }
+
+    public func binPath(
+        for config: BuildConfiguration,
+        scratchPath: [String] = [".build"],
+        triple: String? = nil,
+    ) throws -> [String] {
+        let suffix: String
+
+        #if os(Linux)
```

Comment:
I know this is based on existing code, but you really want to avoid ifdefs in build system code as much as possible in order to avoid unintentionally breaking cross-compilation scenarios -- this depends on the build target, not the host OS. I'd suggest adding a helper method which deduces this suffix from the triple, then use that here.

Reply:
This is in the `_InternalTestSupport` target, which should only be a test target dependency.  As such, we can update this code when we add cross-compilation automated tests.